yt_dlp/extractor/trovo.py

```diff
@@ -163,7 +163,31 @@ def _real_extract(self, url):
         category = vod_info.get('categoryName')
         get_count = lambda x: int_or_none(vod_info.get(x + 'Num'))
 
-        comment_list = try_get(resp, lambda x: x[1]['data']['getCommentList']['commentList'], list) or []
+        comment_list = []
+        for page in itertools.count(1):
+            comments_json = self._call_api(vid, data={
+                'operationName': 'getCommentList',
+                'variables': {
+                    'params': {
+                        'appInfo': {
+                            'postID': vid,
+                        },
+                        'preview': {},
+                        'pageSize': 99,
+                        'page': page,
+                    },
+                },
+                'extensions': {
+                    'persistedQuery': {
+                        'version': 1,
+                        'sha256Hash': 'be8e5f9522ddac7f7c604c0d284fd22481813263580849926c4c66fb767eed25',
+                    },
+                },
+            })
+            comment_list += comments_json['commentList']
+            if comments_json['lastPage']:
+                break
+
```

Comment:
Since comment extraction needs extra requests, move it into `_get_comments` and return `'__post_extractor': self.extract_comments(...)`

Reply:
That seems to break the comments mincount test, should I just remove it?